raiden/tests/smart_contracts/test_netting_channel.py

```diff
@@ -387,6 +387,46 @@ def test_close_called_multiple_times(tester_state, tester_nettingcontracts):
     assert nettingchannel.closingAddress(sender=pkey0) == encode_hex(address0)
 
 
+@pytest.mark.xfail(reason='Issue: #292')
+def test_close_valid_tranfer_different_token(
+        tester_state,
+        tester_nettingcontracts,
+        token_amount,
+        tester_events):
+    """ Valid messages from a different chanel must be rejected. """
+    pkey0, pkey1, nettingchannel = tester_nettingcontracts[0]
+
+    from raiden.tests.fixtures.tester import (
```

Comment:
because it's a fixture, if I imported in the top-level the linter would complain about tests that overwrite the function, also this is used on this test only